### PR TITLE
Support image-based R&C for Paper LPAs

### PR DESCRIPTION
**Pull request by gregtyler**
- Alter schema to change fields available based on form channel
- Add `FileUpload` and `File` struct types
  - `FileUpload` is used in submissions, `File` is used to pass back
- Add `S3Client.UploadFile` to copy base64-encoded files into S3
- On submission of Paper LPAs, store `FileUpload` objects in S3 and return a reference to them in `File`
- Update the fixtures UI to support uploads

For CTC-146 #minor

### CODE REVIEW

File: internal/objectstore/client.go [hunk 1]
```diff
@@ -38,6 +42,38 @@ func (c *S3Client) Put(ctx context.Context, objectKey string, obj any) error {
...
+			ServerSideEncryption: types.ServerSideEncryptionAwsKms,
+		},
+	)
+
+	if err != nil {
+		return shared.File{}, err
+	}
+
+	hash := sha256.New()
+	hash.Write(imgData)
```
Comment (acsauk) [inline internal/objectstore/client.go:68]: Should we be handling the (potential) error returned here?

File: lambda/create/main.go [hunk 2]
```diff
@@ -104,6 +105,20 @@ func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequ
 	data.Status = shared.LpaStatusProcessing
 	data.UpdatedAt = time.Now()
 
+	if data.Channel == shared.ChannelPaper && len(input.RestrictionsAndConditionsImages) > 0 {
+		data.RestrictionsAndConditionsImages = make([]shared.File, len(input.RestrictionsAndConditionsImages))
+		for i, image := range input.RestrictionsAndConditionsImages {
+			path := fmt.Sprintf("%s/scans/rc_%d_%s", data.Uid, i, image.Filename)
```
Comment (acsauk) [inline lambda/create/main.go:111]: Are we ever likely to get identical filenames?

**Review by acsauk — COMMENTED** [review]
This will be cool to see working 🙌 

A couple of Qs

File: lambda/create/main.go (hunk 2 above)
Comment (gregtyler) [reply lambda/create/main.go:111]: I think the inclusion of `i` should prevent that, since `{uid}/scans/rc_{i}` should itself be a unique identifier.

Comment (acsauk) [reply lambda/create/main.go:111]: Ah, missed that! All good then 👍

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**